CondFormats/RPCObjects/interface/RPCInverseAMCLinkMap.h

```diff
@@ -0,0 +1,34 @@
+#ifndef CondFormats_RPCObjects_RPCInverseAMCLinkMap_h
```

Comment:
is there some functionality required such that replacing this class (and the other similar one below) with a simple typedef does not work?